runtime/metricsview/executor_rewrite_time.go

```diff
@@ -8,9 +8,41 @@ import (
 
 	"github.com/rilldata/rill/runtime/drivers"
 	"github.com/rilldata/rill/runtime/pkg/duration"
+	"github.com/rilldata/rill/runtime/pkg/rilltime"
 	"github.com/rilldata/rill/runtime/pkg/timeutil"
 )
 
+func (e *Executor) GetMinTime(ctx context.Context, colName string) (time.Time, error) {
```

Comment:
1. Can drop the `Get`, so just `MinTime`, similar to `Watermark`.
2. All the executors exported functions should be in `runtime/metricsview/executor.go` file – this probably belongs next to the `Watermark` function.
3. Why does this accept a column name? We currently only support one time dimension for metrics views.

Reply:
Update for 1 and 2. For 3, we are currently reusing the method for `time_range_start` and `time_range_end` which accepts a column name.